### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by AlonsoGuevara** [microsoft/graphrag#914] [Announcements]
## What's Changed
- minor: Implement auto templating API.
- minor: Implement query engine API.
- patch: Fix file dumps using json for non ASCII chars
- patch: Stabilize smoke tests for query context building
- patch: fix query embedding
- patch: fix sort_context & max_tokens params in verb

## New Contributors
* @andresmor-ms made their first contribution in https://github.com/microsoft/graphrag/pull/888
* @benx13 made their first contribution in https://github.com/microsoft/graphrag/pull/907

**Full Changelog**: https://github.com/microsoft/graphrag/compare/v0.2.2...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/graphrag/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by shaoqing404** [comment]
更新了prompt_tune之后增加了setting.ymal需要通过--config进行指定，是否可以请求项目组更新文档（https://microsoft.github.io/graphrag/）？

我之前使用了之前的setting文件，因此没有关注到更改。可否请求针对setting的更新进行说明？如果可以，非常感谢！如果不可用，也非常感谢你！
